go/vt/vtorc/config/config.go

```diff
@@ -192,6 +192,15 @@ var (
 		},
 	)
 
+	allowRecovery = viperutil.Configure(
+		"allow-recovery",
+		viperutil.Options[bool]{
+			FlagName: "allow-recovery",
+			Default:  true,
+			Dynamic:  false,
```

Comment:
Is it intentional to require a process restart to change this? To me, it makes sense to try to make as many flags dynamic as possible going forward because it is so much easier to do them one at a time rather an all at once.